resources/views/devices.blade.php

```diff
@@ -34,6 +35,23 @@ function controlDevice(action, publicDeviceId) {
                     url: "/devices/" + action + "/" + publicDeviceId
                 });
             }
+
+            function confirmBeforeDelete(that) {
+                let deviceName = $(that).data("device-name");
+                let deleteLink = $(that).data("delete-link");
+
+                swal({
+                    title: "Are you sure you want to delete [" + deviceName + "]?",
```

Comment:
I pulled down your pull request and started playing with it.  It works great!  I think I may want to tweak my idea a bit though to make it display better now that I can actually see this working...how about modifying this alert to be the following?

```
title: deviceName,
text: "Are you sure you want to delete this device?",
```

I think this makes it display nicely on both desktop and mobile.

<img width="370" alt="screen shot 2018-10-15 at 18 14 42" src="https://user-images.githubusercontent.com/15161181/46981414-39f05380-d0a6-11e8-868e-672f7d8b8f99.png">